yellowbrick/classifier/classification_report.py

```diff
@@ -79,14 +83,17 @@ class ClassificationReport(ClassificationScoreVisualizer):
         Outer dictionary composed of precision, recall, f1, and support scores with
         inner dictionaries specifiying the values for each class listed.
     """
-    def __init__(self, model, ax=None, classes=None, cmap='YlOrRd', **kwargs):
+    def __init__(self, model, ax=None, classes=None, cmap='YlOrRd',
+                 support=None, **kwargs):
         super(ClassificationReport, self).__init__(
             model, ax=ax, classes=classes, **kwargs
         )
 
         self.cmap = color_sequence(cmap)
         self.cmap.set_under(color=CMAP_UNDERCOLOR)
         self.cmap.set_over(color=CMAP_OVERCOLOR)
+        self.support = support
```

Comment:
I would recommend some validation of the support argument at this point, in case a user passes `support="foo"` or something like that. It can be as simple as:

```python
if support not in {None, True, False, "percent", "count"}:
    raise YellowbrickValueError()  
```

This would be a good candidate for a test case as well. 

```python
def test_invalid_support(self):
    """
    Ensure that bad support arguments raise exception 
    """
    with pytest.raises(YellowbrickValueError, match="bad support argument"):
        ClassificationReport(model, support="foo") 
```

Reply:
Completed! 👍